Dev

klikli19 opened the pull request.

klikli19 committed:
Create README.md

klikli19 committed:
correct test

klikli19 committed:
correct test #2

klikli19 committed:
correct test #2

MarinaGubina committed:
Создала описание проекта

MarinaGubina committed:
Создала описание проекта

MarinaGubina committed:
Подправила описание проекта

MarinaGubina committed:
Подправила описание проекта

Hi0ta committed:
feature-2

klikli19 committed:
done: packages, Shelter, TelegramBotConfiguration + add dependency

klikli19 committed:
Merge pull request #1 from klikli19/feature-4

done: packages, Shelter, TelegramBotConfiguration

klikli19 committed:
add in Shelter getters, setters, equals, hashCode and toString

klikli19 committed:
Merge pull request #5 from klikli19/feature-4

klikli19 committed:
correct #1

IlyaB98 committed:
Added dependencies and greeting, corrected the config file

IlyaB98 committed:
Divided all commands into methods, created a menu

MarinaGubina committed:
Merge branch 'dev' into feature-1

MarinaGubina committed:
Added OwnerDog

MarinaGubina committed:
Added OwnerDog

MarinaGubina committed:
Merge pull request #2 from klikli19/feature-1

klikli19 committed:
Merge remote-tracking branch 'origin/dev' into feature-4

klikli19 committed:
add Dog

klikli19 committed:
Merge pull request #6 from klikli19/feature-4

feature-4

IlyaB98 committed:
Merge branch 'dev' into feature-3

IlyaB98 committed:
Merge pull request #8 from klikli19/feature-3

Feature 3

klikli19 committed:
add package liquibase with changelog-master and script + correct classes

klikli19 committed:
Merge branch 'dev' into feature-4

klikli19 committed:
Merge pull request #9 from klikli19/feature-4

add package liquibase with changelog-master and script + correct classes

IlyaB98 committed:
Created buttons under messages, added command enam

IlyaB98 committed:
Merge branch 'dev' into feature-3

IlyaB98 committed:
Merge pull request #10 from klikli19/feature-3

Created buttons under messages, added command enam

klikli19 committed:
add package constant, correct script, listener and Dog, OwnerDog

klikli19 committed:
Merge pull request #11 from klikli19/feature-4

add package constant, correct script, listener and Dog, OwnerDog

IlyaB98 committed:
Merge remote-tracking branch 'origin/dev' into feature-3

IlyaB98 committed:
Merge remote-tracking branch 'origin/dev' into feature-3

IlyaB98 committed:
Made the menu and corrected the greeting

IlyaB98 committed:
Merge pull request #12 from klikli19/feature-3

Feature 3

klikli19 committed:
add DogStatus, OwnerStatus DogService, OwnerDogService, packages: controller, exception + correct pom.xml

klikli19 committed:
add DogStatus, OwnerStatus DogService, OwnerDogService, packages: controller, exception + correct pom.xml

klikli19 committed:
Merge pull request #13 from klikli19/feature-4

Feature 4

klikli19 committed:
correct update method in controllers and service

IlyaB98 committed:
Created a complete bot template

klikli19 committed:
Merge pull request #14 from klikli19/feature-4

correct update method in controllers and service

IlyaB98 committed:
Merge pull request #15 from klikli19/feature-3

Created a complete bot template

klikli19 committed:
add documenting API and JavaDoc except package model

klikli19 committed:
Merge pull request #16 from klikli19/feature-4

add documenting API and JavaDoc except package model and listener.java

MarinaGubina committed:
Added JavaDoc for entity classes

MarinaGubina committed:
Added JavaDoc for entity classes

MarinaGubina committed:
Add JavaDoc for TelegramBotUpdatesListener

MarinaGubina committed:
Add JavaDoc for TelegramBotUpdatesListener

MarinaGubina committed:
Add JavaDoc for TelegramBotUpdatesListener

MarinaGubina committed:
Merge pull request #17 from klikli19/feature-1

feature 1

klikli19 committed:
add method contactDetails

klikli19 committed:
add method contactDetails

MarinaGubina committed:
sending a safety document to the user

MarinaGubina committed:
sending a safety document to the user

dmitry-bizin commented on pom.xml:
```diff
@@ -14,19 +14,62 @@
         <dependency>
             <groupId>org.springframework.boot</groupId>
             <artifactId>spring-boot-starter-web</artifactId>
+            <version>3.0.4</version>
```
версию лучше не писать, чтобы не было конфликта версий, т.к. у вас родительская версия 2.7.9, а здесь 3.0.4 - высока вероятность конфликта. Лучше просто удалить тег версии и будет взята родительская

klikli19 replied:
Окей, исправим

dmitry-bizin commented on pom.xml:
```diff
@@ -14,19 +14,62 @@
         <dependency>
             <groupId>org.springframework.boot</groupId>
             <artifactId>spring-boot-starter-test</artifactId>
+            <version>3.0.4</version>
```
версию лучше не писать, чтобы не было конфликта версий, т.к. у вас родительская версия 2.7.9, а здесь 3.0.4 - высока вероятность конфликта. Лучше просто удалить тег версии и будет взята родительская

dmitry-bizin commented on pom.xml:
```diff
@@ -14,19 +14,62 @@
+        <dependency>
+            <groupId>org.postgresql</groupId>
+            <artifactId>postgresql</artifactId>
+            <version>42.5.4</version>
```
версию лучше не писать

dmitry-bizin commented on pom.xml:
```diff
@@ -14,19 +14,62 @@
+
+        <dependency>
+            <groupId>org.postgresql</groupId>
+            <artifactId>postgresql</artifactId>
```
зачем второй раз? выше уже подключаете

klikli19 replied:
Окей, спасибо.. это все невнимательность)))

dmitry-bizin commented on pom.xml:
```diff
@@ -14,19 +14,62 @@
+
+        <dependency>
+            <groupId>io.swagger.core.v3</groupId>
+            <artifactId>swagger-annotations</artifactId>
```
эта зависимость не нужна по идее

klikli19 replied:
а у нас же есть контроллеры ... и rest-запросы

dmitry-bizin replied:
да, но вот у вас ниже есть springdoc-openapi-ui, откуда как раз и надо брать openapi аннотации

dmitry-bizin commented on src/main/java/pro/sky/animalshelterbot/constant/Commands.java:
```diff
@@ -0,0 +1,77 @@
+public enum Commands {
+    CONTACT_DETAILS("/contact_details", "Оставить контактные данные", "  "),
+    ADVICES("/advices", "Советы", "   "),
+    LIST_DOCUMENTS("/list_documents", "Список документов", "    "),
```
у вас callbackData в виде пробелов отправляется?) интересное решение, конечно))

dmitry-bizin commented on src/main/java/pro/sky/animalshelterbot/controller/DogController.java:
```diff
@@ -0,0 +1,158 @@
+@RestController
+@RequestMapping("dogs")
+public class DogController {
+    @Autowired
```
инъекцию лучше делать через конструктор, также поле можно сделать будет финальным

dmitry-bizin commented on src/main/java/pro/sky/animalshelterbot/controller/OwnerDogController.java:
```diff
@@ -0,0 +1,160 @@
+@RestController
+@RequestMapping("owners")
+public class OwnerDogController {
+    @Autowired
```
инъекцию лучше делать через конструктор, также поле можно сделать будет финальным

dmitry-bizin commented on src/main/java/pro/sky/animalshelterbot/exception/OwnerDogNotFoundException.java:
```diff
@@ -0,0 +1,17 @@
+ * Исключение наследуется от {@link RuntimeException}
+ * @author Kilikova Anna
+ */
+public class OwnerDogNotFoundException extends RuntimeException{
```
ResponseStatus забыли?

klikli19 replied:
да, поправила

dmitry-bizin commented on src/main/java/pro/sky/animalshelterbot/service/DogService.java:
```diff
@@ -0,0 +1,99 @@
+@Service
+public class DogService {
+
+    @Autowired
```
через конструктор и final

dmitry-bizin commented on src/main/java/pro/sky/animalshelterbot/service/OwnerDogService.java:
```diff
@@ -0,0 +1,98 @@
+
+@Service
+public class OwnerDogService {
+    @Autowired
```
через конструктор и final

dmitry-bizin commented on src/main/java/pro/sky/animalshelterbot/service/TelegramBotUpdatesListener.java:
```diff
@@ -0,0 +1,389 @@
+                telegramBot.execute(new SendMessage(update.callbackQuery().message().chat().id(),
+                        update.callbackQuery().data()));
+                //startMenu
+                if (update.callbackQuery() != null && update.callbackQuery().data().equals(" ")) {
```
в equals лучше передать константу

MarinaGubina committed:
Merge branch 'dev' into feature-1

klikli19 committed:
corrected according to comments

klikli19 committed:
corrected according to comments

alexsa-8 committed:
Merge remote-tracking branch 'origin/feature-2' into dev

alexsa-8 committed:
Начало субъектов и констант

MarinaGubina committed:
add all buttons for the task

MarinaGubina committed:
add all buttons for the task

MarinaGubina committed:
Merge pull request #19 from klikli19/feature-1

editing buttons

klikli19 committed:
Merge remote-tracking branch 'origin/dev' into feature-4

# Conflicts:
#	src/main/java/pro/sky/animalshelterbot/service/TelegramBotUpdatesListener.java

klikli19 committed:
contactDetails method not working

klikli19 committed:
contactDetails method not working

alexsa-8 committed:
Волонтёр и констант с JavaDoc

alexsa-8 committed:
Merge pull request #21 from klikli19/feature-2

создал субъект волонтёр и константу статус волонтёра

klikli19 committed:
add Recommendations files and methods

klikli19 committed:
add Recommendations files and methods

IlyaB98 committed:
Created an autogenerate method callbackData
Attached documents to messages:
List documents
Advices cynologists
List cynologists
Reasons refusal

IlyaB98 committed:
Created an autogenerate method callbackData
Attached documents to messages:
List documents
Advices cynologists
List cynologists
Reasons refusal

IlyaB98 committed:
Merge pull request #22 from klikli19/feature-3

Created an autogenerate method callbackData

klikli19 committed:
add shelterData method + map.jpg

klikli19 committed:
add shelterData method + map.jpg

klikli19 committed:
Merge branch 'dev' into feature-4

klikli19 committed:
Merge pull request #18 from klikli19/feature-4

Feature 4

alexsa-8 committed:
Внёс данные волонтёра в sql

IlyaB98 committed:
Created a method for saving contact data in the database

IlyaB98 committed:
Created a method for saving contact data in the database

IlyaB98 committed:
Merge branch 'dev' into feature-3

IlyaB98 committed:
Merge pull request #23 from klikli19/feature-3

Created a method for saving contact data in the database

klikli19 committed:
correct TelegramBotUpdatesListener, OwnerDog, Shelter
 and add changeset:4

klikli19 committed:
correct TelegramBotUpdatesListener, OwnerDog, Shelter
 and add changeset:4

klikli19 committed:
Merge pull request #24 from klikli19/feature-4

correct TelegramBotUpdatesListener, OwnerDog, Shelter and add changeset

alexsa-8 committed:
Merge remote-tracking branch 'origin/dev' into feature-2

# Conflicts:
#	src/main/resources/liquibase/202303291500_dog_shelter_table.sql

alexsa-8 committed:
I have entered the volunteer's data in sql

alexsa-8 committed:
I have entered the volunteer's data in sql

IlyaB98 committed:
Rewrote the handling of user requests for the switch

IlyaB98 committed:
Rewrote the handling of user requests for the switch

IlyaB98 committed:
Merge remote-tracking branch 'origin/feature-3' into feature-3

IlyaB98 committed:
Redesigned the application architecture

IlyaB98 committed:
Redesigned the application architecture

IlyaB98 committed:
Merge branch 'dev' into feature-3

IlyaB98 committed:
Merge pull request #25 from klikli19/feature-3

Feature 3

MarinaGubina committed:
add report: model, enum status, repository,service

MarinaGubina committed:
add report: model, enum status, repository,service

MarinaGubina committed:
create table "report" in database

MarinaGubina committed:
create table "report" in database

MarinaGubina committed:
add ReportController

MarinaGubina committed:
add ReportController

MarinaGubina committed:
Merge pull request #26 from klikli19/feature-1

Added report

klikli19 committed:
added fields to Report, add changeset:5, add method download report

klikli19 committed:
added fields to Report, add changeset:5, add method download report

alexsa-8 committed:
created a Volunteer Service, Volunteer Repository, fixed sql

alexsa-8 committed:
created a Volunteer Service, Volunteer Repository, fixed sql

klikli19 committed:
correct#2

alexsa-8 committed:
created a Volunteer Controller, fixed sql

MarinaGubina committed:
add downloadReport

MarinaGubina committed:
add downloadReport

MarinaGubina committed:
sending the report correctly

MarinaGubina committed:
sending the report correctly

MarinaGubina committed:
Merge pull request #29 from klikli19/feature-1

Save report to database

klikli19 committed:
Merge remote-tracking branch 'origin/dev' into feature-4

# Conflicts:
#	src/main/java/pro/sky/animalshelterbot/entity/Report.java
#	src/main/java/pro/sky/animalshelterbot/service/ReportService.java
#	src/main/java/pro/sky/animalshelterbot/service/SendReportMenuService.java
#	src/main/resources/liquibase/202303291500_dog_shelter_table.sql

BizinMitya commented on src/main/java/pro/sky/animalshelterbot/entity/Dog.java:
```diff
@@ -0,0 +1,138 @@
+    @Enumerated(EnumType.STRING)
+    @Column(name = "status")
+    private PetStatus status;
+
```
почему собака не привяза к приюту?

alexsa-8 committed:
fixed the VolunteerController

alexsa-8 committed:
fixed the VolunteerController

alexsa-8 committed:
Merge remote-tracking branch 'origin/dev' into feature-2

# Conflicts:
#	src/main/java/pro/sky/animalshelterbot/entity/Report.java
#	src/main/java/pro/sky/animalshelterbot/entity/Volunteer.java

IlyaB98 committed:
Wrote tests for dog controller and dog owner controller

IlyaB98 committed:
Wrote tests for dog controller and dog owner controller

IlyaB98 committed:
Merge pull request #30 from klikli19/feature-3

Wrote tests for dog controller and dog owner controller

alexsa-8 committed:
Merge remote-tracking branch 'origin/dev' into feature-2

alexsa-8 committed:
correction in the VolunteerService

alexsa-8 committed:
correction in the VolunteerService

dmitry-bizin commented on src/main/java/pro/sky/animalshelterbot/constant/Commands.java:
```diff
@@ -0,0 +1,119 @@
+     */
+    public static String generateCallbackData() {
+        StringBuilder sb = new StringBuilder(" ");
+        sb.append(" ".repeat(Math.max(0, count + 1)));
```
Интересно вы придумали конечно...)

alexsa-8 committed:
Merge remote-tracking branch 'origin/feature-2' into feature-2

# Conflicts:
#	src/main/java/pro/sky/animalshelterbot/entity/OwnerDog.java
#	src/main/java/pro/sky/animalshelterbot/entity/Volunteer.java
#	src/main/java/pro/sky/animalshelterbot/service/TelegramBotUpdatesListener.java
#	src/main/resources/liquibase/202303291500_dog_shelter_table.sql

klikli19 committed:
Merge remote-tracking branch 'origin/dev' into feature-4

alexsa-8 committed:
Merge pull request #31 from klikli19/feature-2

Feature 2

klikli19 committed:
Merge remote-tracking branch 'origin/dev' into feature-4

alexsa-8 committed:
Report. In the ProcessMessageService, I replaced the "ДА" button with "Кошку" and "Собаку".

MarinaGubina committed:
corrected the regex

klikli19 committed:
add Scheduler

MarinaGubina committed:
corrected the processing of updates

IlyaB98 committed:
Divided the menu by shelters

IlyaB98 committed:
Added a logger to the methods and filled in the dating rules

IlyaB98 committed:
Merge pull request #33 from klikli19/feature-3

Feature 3

klikli19 committed:
correct Scheduler

alexsa-8 committed:
VolunteerController, Volunteer.

alexsa-8 committed:
VolunteerController, Volunteer.

klikli19 committed:
add method change days of reports

MarinaGubina committed:
added ReportServiceTest

klikli19 committed:
Merge pull request #28 from klikli19/feature-4

Feature 4

MarinaGubina committed:
corrected method delete in ReportServiceTest

MarinaGubina committed:
Merge branch 'dev' into feature-1

# Conflicts:
#	src/main/java/pro/sky/animalshelterbot/entity/Report.java
#	src/main/java/pro/sky/animalshelterbot/service/SendReportMenuService.java
#	src/main/java/pro/sky/animalshelterbot/service/TelegramBotUpdatesListener.java

MarinaGubina committed:
merge dev into feature-1

MarinaGubina committed:
Merge pull request #32 from klikli19/feature-1

feature-1

alexsa-8 committed:
Merge branch 'dev' into feature-2

# Conflicts:
#	src/main/java/pro/sky/animalshelterbot/entity/Report.java
#	src/main/java/pro/sky/animalshelterbot/service/ProcessMessageService.java

MarinaGubina committed:
corrected save contact

MarinaGubina committed:
added Cat, OwnerCat

MarinaGubina committed:
corrected call volunteer

MarinaGubina committed:
Merge pull request #34 from klikli19/feature-1

feature-1

MarinaGubina committed:
corrected liquibase

MarinaGubina committed:
added all classes with cat

MarinaGubina committed:
corrected save contact in database

MarinaGubina committed:
Merge pull request #35 from klikli19/feature-1

feature1

klikli19 committed:
add method change days of probation, correct change status owners

klikli19 committed:
add method notice to Owners about probation, add NumberNotFoundException

alexsa-8 committed:
Entered the data of the cat shelter

alexsa-8 committed:
Entered the data of the cat shelter

klikli19 committed:
correct notifications owners, add ProbationNotSpecifiedException, logger, Manual.pdf

klikli19 committed:
correct logger

klikli19 committed:
correct#2 logger

klikli19 committed:
Merge pull request #36 from klikli19/feature-4

add method change days of probation, correct change status owners

alexsa-8 committed:
Merge remote-tracking branch 'origin/feature-2' into feature-2

klikli19 committed:
add recommendations for cats

klikli19 committed:
Merge pull request #39 from klikli19/feature-4

Feature-4

alexsa-8 committed:
Entered the data of the cat shelter

alexsa-8 committed:
Entered the data of the cat shelter

alexsa-8 committed:
Merge remote-tracking branch 'origin/dev' into feature-2

alexsa-8 committed:
Added directions to the cat shelter

alexsa-8 committed:
Added to toString chatId

alexsa-8 committed:
Added to the 103 line mapCatShelter.jpg

alexsa-8 committed:
In Commands, ShelterDataMenuService and processCallbackQuery added the pass clearance method

IlyaB98 committed:
Added edits to the list of documents, safety recommendations and rules for dating in a cat shelter

IlyaB98 committed:
Merge pull request #40 from klikli19/feature-3

Added edits to the list of documents, safety recommendations and rule…

MarinaGubina committed:
added reportCat

MarinaGubina committed:
Merge branch 'dev' into feature-1

MarinaGubina committed:
Merge pull request #41 from klikli19/feature-1

feature-1

klikli19 committed:
correct Report

klikli19 committed:
Merge pull request #38 from klikli19/feature-2

Feature 2

alexsa-8 committed:
A method for selecting a pet has been added to Commands, ShelterDataMenuService and processCallbackQuery

alexsa-8 committed:
A method for selecting a pet has been added to Commands, ShelterDataMenuService and processCallbackQuery

alexsa-8 committed:
A method for selecting a pet has been added to Commands, ShelterDataMenuService and processCallbackQuery

klikli19 committed:
correct Scheduled

klikli19 committed:
Merge pull request #43 from klikli19/feature-4

correct Scheduled

MarinaGubina committed:
added test for Cat classes(service,controller)

alexsa-8 committed:
Merge remote-tracking branch 'origin/dev' into feature-2

klikli19 committed:
tests1

alexsa-8 committed:
did a test on VolunteerService

alexsa-8 committed:
did a test on VolunteerService

alexsa-8 committed:
Merge pull request #42 from klikli19/feature-2

Feature 2

alexsa-8 committed:
Added emoticons

klikli19 committed:
tests2

klikli19 committed:
tests3

klikli19 committed:
tied shelter and animal

MarinaGubina committed:
Merge remote-tracking branch 'origin/dev' into feature-1

MarinaGubina committed:
Merge pull request #44 from klikli19/feature-1

feature 1 - testing

MarinaGubina committed:
test for AdvicesMenuService and ReportDogController

IlyaB98 committed:
Made a multiplayer mode, wrote tests for cats and dogs

IlyaB98 committed:
Merge remote-tracking branch 'origin/dev' into feature-3

# Conflicts:
#	src/main/java/pro/sky/animalshelterbot/service/ProcessCallbackQueryService.java
#	src/main/resources/liquibase/202303291500_dog_shelter_table.sql
#	src/test/java/pro/sky/animalshelterbot/controller/DogControllerTests.java
#	src/test/java/pro/sky/animalshelterbot/controller/OwnerDogControllerTests.java

IlyaB98 committed:
Merge pull request #47 from klikli19/feature-3

Made a multiplayer mode, wrote tests for cats and dogs

IlyaB98 committed:
bug fix

IlyaB98 committed:
Merge pull request #48 from klikli19/feature-3

bug fix

alexsa-8 committed:
Merge remote-tracking branch 'origin/dev' into feature-2

klikli19 committed:
Merge branch 'dev' into feature-4

klikli19 committed:
Merge pull request #45 from klikli19/feature-4

tests1

klikli19 committed:
Merge pull request #46 from klikli19/feature-2

Added emoticons

dmitry-bizin commented on src/main/java/pro/sky/animalshelterbot/controller/VolunteerController.java:
```diff
@@ -0,0 +1,169 @@
+@RestController
+@RequestMapping("volunteer")
+public class VolunteerController {
+    private VolunteerService service;
```
final

dmitry-bizin commented on src/main/java/pro/sky/animalshelterbot/controller/VolunteerController.java:
```diff
@@ -0,0 +1,169 @@
+        this.service = service;
+    }
+
+    public VolunteerController() {
```
конструктор не нужен без параметров

dmitry-bizin commented on src/main/java/pro/sky/animalshelterbot/entity/Volunteer.java:
```diff
@@ -60,9 +63,26 @@ public Volunteer(Long id, String name, String phone, VolunteerStatus status) {
     /**
      * Конструктор без параметров для создания объекта "волонтёр"
      */
+    public Volunteer(Long chatId, String name, String phone) {
```
как же без параметров, если есть параметры?)

BizinMitya commented on src/main/java/pro/sky/animalshelterbot/repository/ReportCatRepository.java:
```diff
@@ -0,0 +1,22 @@
+ */
+public interface ReportCatRepository extends JpaRepository<ReportCat, Long> {
+
+    @Query(value="select distinct  on (owner_cat_id) * from report_cat order by owner_cat_id, date_message desc ", nativeQuery = true)
```
а что это за запрос такой? он не будет работать, здесь ошибка SQL

BizinMitya commented on src/main/java/pro/sky/animalshelterbot/repository/ReportCatRepository.java:
```diff
@@ -0,0 +1,22 @@
+    @Query(value="select distinct  on (owner_cat_id) * from report_cat order by owner_cat_id, date_message desc ", nativeQuery = true)
+    Collection<ReportCat> findReports();
+
+    @Query(value="select distinct  on (owner_cat_id) * from report_cat order by owner_cat_id, date_message ", nativeQuery = true)
```
а что это за запрос такой? он не будет работать, здесь ошибка SQL

dmitry-bizin commented on src/main/java/pro/sky/animalshelterbot/service/RecommendationMenuService.java:
```diff
@@ -68,61 +83,131 @@ public SendMessage recommendationMenu(Update update) {
+        logger.info("Launched method: recommendations_transportation, for user with id: " +
+                update.callbackQuery().message().chat().id());
+
+        String pathDog = "src/main/resources/recommendations/Recommendations_of_Transportation.pdf";
```
можно вот [так](https://www.baeldung.com/spring-classpath-file-access#2-using-value) получать ресурсы, будет более красивее)

alexsa-8 committed:
Merge remote-tracking branch 'origin/dev' into feature-2

alexsa-8 committed:
wrote a VolunteerControllerTest

alexsa-8 committed:
Merge pull request #49 from klikli19/feature-2

Feature 2

klikli19 committed:
tests

klikli19 committed:
Merge remote-tracking branch 'origin/dev' into feature-4

alexsa-8 committed:
fixed the VolunteerServiceTest

alexsa-8 committed:
Merge remote-tracking branch 'origin/dev' into feature-2

klikli19 committed:
tests#2

klikli19 committed:
tests#2

klikli19 committed:
correct

MarinaGubina committed:
Merge branch 'dev' into feature-1

MarinaGubina committed:
test DogController

klikli19 committed:
correct

klikli19 committed:
Merge pull request #50 from klikli19/feature-4

tests

klikli19 committed:
correct tests service

klikli19 committed:
Merge pull request #51 from klikli19/feature-4

correct tests service

alexsa-8 committed:
Merge remote-tracking branch 'origin/dev' into feature-2

klikli19 committed:
add report controller test

MarinaGubina committed:
test reportCat

MarinaGubina committed:
test

IlyaB98 committed:
fix save contact in database

IlyaB98 committed:
fix save contact in database

IlyaB98 committed:
Merge pull request #54 from klikli19/feature-3

fix save contact in database

MarinaGubina committed:
Merge branch 'dev' into feature-1

MarinaGubina committed:
update Readme.md

MarinaGubina committed:
update Readme.md

MarinaGubina committed:
test AdvicesMenuService

MarinaGubina committed:
Merge pull request #53 from klikli19/feature-1

testing

klikli19 committed:
Merge remote-tracking branch 'origin/dev' into feature-4

# Conflicts:
#	src/test/java/pro/sky/animalshelterbot/controller/ReportCatControllerTest.java

alexsa-8 committed:
ReportDogControllerTest

alexsa-8 committed:
Merge pull request #55 from klikli19/feature-2

Feature 2

klikli19 committed:
correct report service + owner service

klikli19 committed:
Merge branch 'dev' into feature-4

klikli19 committed:
Merge pull request #56 from klikli19/feature-4

Feature 4

klikli19 committed:
correct report service + owner service

klikli19 committed:
Merge remote-tracking branch 'origin/feature-4' into feature-4

klikli19 committed:
Merge pull request #57 from klikli19/feature-4

Feature 4

MarinaGubina committed:
test AdvicesMenuService new

MarinaGubina committed:
Merge branch 'dev' into feature-1

MarinaGubina committed:
corrected test for Reports classes

MarinaGubina committed:
test for ShelterDataMenuService

MarinaGubina committed:
Merge pull request #58 from klikli19/feature-1

tests
